pom.xml

```diff
@@ -555,6 +555,104 @@
+
+                            <artifact>com.hazelcast:hazelcast:3.8.3</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.8.3:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.8.4</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.8.4:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.8.5</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.8.5:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.8.6</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.8.6:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.8.7</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.8.7:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.8.8</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.8.8:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.8.9</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.8.9:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.9</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.9:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.9.1</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.9.1:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.9.2</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.9.2:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.9.3</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.9.3:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.9.4</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.9.4:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.10</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.10:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.10.1</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.10.1:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.10.2</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.10.2:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.10.3</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.10.3:jar:tests</artifact>
+
+                            <artifact>com.hazelcast:hazelcast:3.10.4</artifact>
+                            <artifact>com.hazelcast:hazelcast:3.10.4:jar:tests</artifact>
```

Comment:
These come from `com.hazelcast.test.starter.test.PatchLevelCompatibilityTest`, right? 
The test is ignored (and has been for more than 3 years)
@vbekiaris do we actually need that test? Should we rather remove that one? It would leave us with just a few jars.

Reply:
Yes coming from PatchLevelCompatibilityTest. I did not see the ignored annotation. I will delete the versions in pom.xml

Reply:
> do we actually need that test? 

I'd say no, let's get rid of the test altogether